samcli/commands/build/build_context.py

```diff
@@ -24,20 +24,20 @@ class BuildContext:
     _BUILD_DIR_PERMISSIONS = 0o755
 
     def __init__(
-            self,
```

Comment:
Why is the formatting different?

Reply:
Well as per @aahung 

>I just ran black on that file, it does not do anything, then I made a small change, it reformatted it

It seems `black` isn't yet stable. 😩😩😩

Great PR otherwise, approved.